### PR TITLE
Run rules linter in master and PRs 

### PR DESCRIPTION
**Pull request by Ana06**
Create a CI GitHub action with a `rule_linter` job similar to the one in [capa](https://github.com/fireeye/capa/pull/166).

### CODE REVIEW

**Comment by Ana06** [comment]
GitHub Actions are not enabled in this repo 😭 😭 😭 @williballenthin @mr-tz can you please enable it? 🙏

**Comment by Ana06** [comment]
The linter fails because of the bug being solved in https://github.com/fireeye/capa/pull/166

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    # We check the submodules separately as the rules submodule's reference may not be our PR/master
+    - name: Checkout capa without submodules
+      uses: actions/checkout@v2
+      with:
+        repository: fireeye/capa
+    - name: Checkout capa-rules
+      uses: actions/checkout@v2
+      with:
+        path: rules
+    - name: Checkout capa-rules
```
Comment (mr-tz) [inline .github/workflows/tests.yml:22]: This should be "Checkout capa-testfiles"?

Comment (Ana06) [reply .github/workflows/tests.yml:22]: copy-paste... 🙈

**Comment by Ana06** [comment]
The syncronization of the submodule seems to be working properly: https://github.com/fireeye/capa/commit/96453db3be05c784f047492eeefd6f3ee299e917 🥳

**Pull request merged**